### PR TITLE
Add documentation of the controller.daemonset.useHostPort option

### PR DESCRIPTION
**Pull request by Miouge1**
**What this PR does / why we need it**:

`controller.daemonset.useHostPort` is not documented.

**Which issue this PR fixes** 

Add documentation for `controller.daemonset.useHostPort`

### CODE REVIEW

**Comment by Miouge1** [comment]
Do I really need to increase the chart version to add doc?

**Comment by timstoop** [comment]
Yes, all changes to a chart, even in the docs, requires a bump of the version.

**Comment by Miouge1** [comment]
OK. Done!

File: stable/nginx-ingress/README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Parameter | Description | Default
 `controller.scope.namespace` | namespace to watch for ingress | `""` (use the release namespace)
 `controller.extraArgs` | Additional controller container arguments | `{}`
 `controller.kind` | install as Deployment or DaemonSet | `Deployment`
+`controller.daemonset.useHostPort` | If `controller.kind` is `DaemonSet`, this will enable enable `hostPort` for TCP/80 and TCP/443 | false
```
Comment (unguiculus) [inline stable/nginx-ingress/README.md]: double enable: `enable enable`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/nginx-ingress/README.md (hunk 1 above)
Comment (Miouge1) [reply stable/nginx-ingress/README.md]: Oups :) fixed that and rebased onto master to solve merge conflict.

**Comment by Miouge1** [comment]
@unguiculus it should be fixed in latest commit. Can you have a look?

**Comment by timstoop** [comment]
/assign @jackzampolin 
/assign @mgoodness 
/assign @chancez 
Due to being the maintainers of the Chart!

**Review by unguiculus — APPROVED** [review]

**Pull request merged**